src/org/yccheok/jstock/engine/Index.java

```diff
@@ -56,7 +56,8 @@
     TWII(Country.Taiwan, "TSEC", "Taiwan Stock Exchange Weighted Index", Code.newInstance("^TWII")),
     FTSE(Country.UnitedKingdom, "FTSE 100", "FTSE 100 Index", Code.newInstance("^FTSE")),
     DJI(Country.UnitedState, "DOW JONES", "Dow Jones Industrial Average", Code.newInstance("^DJI")),
-    IXIC(Country.UnitedState, "NASDAQ", "NASDAQ Composite Index", Code.newInstance("^IXIC"));
+    IXIC(Country.UnitedState, "NASDAQ", "NASDAQ Composite Index", Code.newInstance("^IXIC")),
+    SP500(Country.UnitedState, "S&P500", "Standard and Poors 500 Index", Code.newInstance("^GSPC"));
```

Comment:
Any reason you feel S&P500 should be included in United States stock index? I had received request once a while to add this index into United States. However, I'm still reluctant to add it in. I'm believe in "less is more". I only want to show info to users, which really matter to them. Currently, we have Dow Jones and Nasdaq, does 90% of United States users like to have S&P too? (I'm not from United States. Hence, my view doesn't represent local)